plugins/provisioners/ansible/cap/guest/pip/pip.rb

```diff
@@ -18,9 +20,20 @@ def self.pip_install(machine, package = "", version = "", pip_args = "", upgrade
             machine.communicate.sudo "pip install #{args_array.join(' ')}"
           end
 
-          def self.get_pip(machine)
+          def self.get_pip(machine, pip_install_cmd=DEFAULT_PIP_INSTALL_CMD)
+
+            # The objective here is to get pip either by default
+            # or by the argument passed in. The objective is not 
+            # to circumvent the pip setup by passing in nothing.
+            # Thus, we stick with the default on an empty string.
+            # Typecast added in the check for safety.
+
+            if pip_install_cmd == pip_install_cmd.to_s.empty?
```

Comment:
This conditional should just be:

```ruby
if pip_install_cmd.to_s.empty?
```

Reply:
Thanks so much for catching that Sir! That was silly of me. You're absolutely right. I just made the update locally and re-ran the tests and they passed. I will commit. 
![tests-passing-locally](https://user-images.githubusercontent.com/10778616/52883649-56be9580-3131-11e9-8a41-060d6363952e.PNG)